lib/Target/IQM/TranslateToIQMJson.cpp

```diff
@@ -21,16 +21,17 @@ namespace cudaq {
 
 static LogicalResult emitOperation(nlohmann::json &json, Emitter &emitter,
                                    Operation &op);
+
 static LogicalResult emitEntryPoint(nlohmann::json &json, Emitter &emitter,
-                                    qtx::CircuitOp circuitOp) {
-  if (circuitOp.getBody().getBlocks().size() != 1)
-    circuitOp.emitError("Cannot map qtx Circuit op with more than 1 block to "
-                        "IQM Json. Must be a flat circuit representation.");
+                                    func::FuncOp op) {
+  if (op.getBody().getBlocks().size() != 1)
+    op.emitError("Cannot translate kernels with more than 1 block to IQM Json. "
+                 "Must be a flat representation.");
```

Comment:
```suggestion
                 "Must be a straight-line representation.");
```
I'm thinking this means no special control flow (ops with regions) as well.